doc/source/digital-ocean/step-zero-digital-ocean.rst

```diff
@@ -0,0 +1,78 @@
+.. _digital-ocean:
+
+Step Zero: Kubernetes on Digital Ocean
+--------------------------------------
+
+You can create a Kubernetes cluster `either through the Digital Ocean website, or using the Digital Ocean command line tools <https://www.digitalocean.com/>`_.
+
+This page describes the commands required to setup a Kubernetes cluster using the command line.
+If you prefer to use the Digital Ocean portal see the `Digital Ocean Get Started <https://www.digitalocean.com/products/kubernetes>`_.
+
+
+#. Prepare your Digital Ocean shell environment.
+   
+   a. **Install command-line tools locally**. You'll need at least v1.13.0. 
+
+     You can either follow the `installation instructions <https://github.com/digitalocean/doctl/blob/master/README.md>` or use the commands below:
```

Comment:
Let's correct the indentation for the a/b/c. See `View file` for a better view of this.